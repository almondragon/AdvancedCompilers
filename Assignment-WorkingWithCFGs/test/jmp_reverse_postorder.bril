@main {
#ARGS: -p
  v: int = const 4;
  jmp .somewhere;
  v: int = const 2;
.somewhere:
  print v;
}
