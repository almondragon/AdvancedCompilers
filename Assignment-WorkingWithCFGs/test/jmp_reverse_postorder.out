This is the reverse order:  ['b0', 'somewhere']
